### ISSUE TITLE
Cannot override layout.position

### ISSUE THREAD

**Issue by felipecrs** [dermotduffy/frigate-hass-card#834]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:** 4.0.0

**Last working release (if known):** N/A

**Browser and Operating System:** N/A

**Description of problem:**

By default I want my card to have the following position:

```yaml
layout:
  fit: cover
  position:
    x: 0
    'y': 0
```

But when in full screen, I want:

```yaml
layout:
  fit: contain
  position:
    x: null
    'y': null
```

But the validation does not let me proceed:

https://user-images.githubusercontent.com/29582865/191889202-06f8b2d4-3418-49d9-8800-2e3f4ca7c39e.mp4

**Javascript errors shown in the web inspector (if applicable):**

N/A

**Additional information:**

N/A

**Comment by dermotduffy** [comment]
Why do you want the fields to be null, i.e. what behavior are you looking for when you are trying to set null? I would have assumed default is equivalent to `x: 0` and `y: 0`. The input validation only accepts numbers, the values provided are used to set [object-position](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position).

**Comment by felipecrs** [comment]
The default comes with no value for these fields, which means the object will be centered. This is what happens if I keep them with 0:

![Screenshot_20220923-010923_Home Assistant.jpg](https://user-images.githubusercontent.com/29582865/191890605-2443d65e-e8ce-48c1-a7d6-64af945c3455.jpg)

They don't get centered anymore.

I worked around inverting the condition (i.e. setting the default for full screen and overriding when not in full screen).

**Comment by dermotduffy** [comment]
What about just setting them to 50? Does that not center?

**Comment by felipecrs** [comment]
That seems to work. Thank you! But, for some very weird reason, it does not work for just one of my cards:


https://user-images.githubusercontent.com/29582865/191966697-2033e84f-fc80-4038-9dd1-7bafc2f86684.mp4

**Comment by dermotduffy** [comment]
> But, for some very weird reason, it does not work for just one of my cards:

Sorry, I can't quite tell what you expect it to do :--( Is it that it's not vertically centering the camera in fullscreen mode, perhaps?

**Comment by felipecrs** [comment]
Yes, exactly

**Comment by dermotduffy** [comment]
@felipecrs Would you mind trying out https://github.com/dermotduffy/frigate-hass-card/pull/851 ? It's already merged on the `release-4.1.0` branch. It fixes a different bug I could reproduce, but I'm hoping it has also fixed this one (that I could not reproduce).

Mind giving it a try?

**Comment by felipecrs** [comment]
Awesome, it fixed the issue for me. Thanks a lot!
